src/coreclr/pal/src/map/virtual.cpp

```diff
@@ -2157,6 +2166,9 @@ void ExecutableMemoryAllocator::TryReserveInitialMemory()
     // (thus avoiding reserving memory below 4GB; besides some operating systems do not allow that).
     // If libcoreclr is loaded at high addresses then try to reserve memory below its location.
     coreclrLoadAddress = (UINT_PTR)PAL_GetSymbolModuleBase((void*)VirtualAlloc);
+
+#ifdef TARGET_XARCH
+    const int32_t MemoryProbingIncrement = 128 * 1024 * 1024;
```

Comment:
Unrelated to this PR: it looks like current logic for x64 might end up allocating just 128Mb for the initial memory in the worst case

Reply:
Not really a problem, better than nothing indeed